com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Camera/HDCameraUI.PhysicalCamera.Drawers.cs

```diff
@@ -9,7 +9,7 @@ namespace UnityEditor.Rendering.HighDefinition
 
     static partial class HDCameraUI
     {
-        partial class PhysicalCamera
+        public partial class PhysicalCamera
```

Comment:
If you change this to be `public` I would change the `partial` declaratio too on the file `.Skin,cs`

Reply:
what would it bring? since i only need internal visibility access to the DrawerPreset

Reply:
is just to mantain coherence between partial declarations, that's it

Reply:
done in next commit